features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```
@@ -1571,6 +1571,9 @@
 
   "on_demand_config.on_initial_use.enable_sms_otp_password_recovery_if_connector_enabled": false,
 
+  "actions.http_connections.read_timeout": 5000,
```

```suggestion
  "actions.api_client.http_connections.read_timeout": 5000,
```
Verify with @malithie as well.

---

Let's keep it as http connections. http is a specific mechanism we connect. Maybe in future we might either give the same extension point over a different mechanism like a script, interface etc. Still we have a way to differentiate

---

Suggested a different convention